ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala

```diff
@@ -95,18 +99,20 @@ final class FtpSettings private (
       credentials: FtpCredentials = credentials,
       binary: Boolean = binary,
       passiveMode: Boolean = passiveMode,
-      configureConnection: FTPClient => Unit = configureConnection
+      configureConnection: FTPClient => Unit = configureConnection,
+      proxy: Option[Proxy] = None
   ): FtpSettings = new FtpSettings(
     host = host,
     port = port,
     credentials = credentials,
     binary = binary,
     passiveMode = passiveMode,
-    configureConnection = configureConnection
+    configureConnection = configureConnection,
+    proxy = proxy
   )
 
   override def toString =
-    s"""FtpSettings(host=$host,port=$port,credentials=$credentials,binary=$binary,passiveMode=$passiveMode,configureConnection=$configureConnection)"""
+    s"""FtpSettings(host=$host,port=$port,credentials=$credentials,binary=$binary,passiveMode=$passiveMode,configureConnection=$configureConnection),proxy=$proxy"""
```

Comment:
The `)` should move to the end. You may make this a single-" string and split it with one field per line, so this doesn't happen again.